src/locales/en/translation.json

```diff
@@ -857,8 +857,11 @@
         "subHeading": "If you’ve already been to the Post Office",
         "paragraph3": "Your identity check is still being processed.",
         "paragraph4": "You’ll get an email when there’s a result for your identity check - usually within a day of going to the Post Office.",
-        "subHeading2": "If you need help",
-        "paragraph5": "Contact us if you:",
+        "subHeading2": "If you want to prove your identity another way",
+        "paragraph5": "If you no longer want to prove your identity at a Post Office, you can <a class=\"govuk-link\" href=\"/ipv/page/pyi-f2f-delete-details\">delete your details and try to prove your identity another way</a>.",
```

Comment:
I think this needs to submit a journey action, rather than just linking to the page - I think it should be something like `/ipv/journey/page-ipv-pending/next`

(There are some similar examples from COI, e.g. on page-ipv-reuse)